docs-source/spring/content/development/cli/_index.md

```diff
@@ -141,10 +141,9 @@ The following is an example development workflow using the CLI:
    application/namespace created successfully and image pull secret (registry-auth) created successfully and database TNSAdmin/wallet secret created successfully
    ```
 
-
 3. Use the `delete` command to delete an application namespace (Kubernetes *namespace*) completely or a specific Microservice inside an application namespace.
 
-    > ATTENTION: <ins>**Be carefully**</ins> because it is not possible rollback the components deleted.
+    > ATTENTION: **Be careful** because it is not possible rollback the components deleted.
```

Comment:
ATTENTION: Ensure that you want to completely delete the application namespace. You cannot rollback the components once deleted.